RecoLocalTracker/SiStripClusterizer/plugins/ClustersFromRawProducer.cc

```diff
@@ -444,7 +443,9 @@ void ClusterFiller::fill(StripClusterizerAlgorithm::output_t::FastFiller & recor
   if(!record.empty()) incNoZ();
 
   // COUT << "filled " << record.size() << std::endl;
-  
+} catch (edmNew::CapacityExaustedException) {
```

Comment:
do not think so, will verify.
I run 100 events throwing at each det and run to completion